### ISSUE TITLE
Errors in Cell after generating SDL

### ISSUE THREAD

**Issue by Tobbe** [redwoodjs/redwood#4766]
# Issue

When generating first the SDL for a model, and then a Cell for the same model you'll get an error printed to the console, and the generated code will be wrong.

# Reproduction

```
yarn create redwood-app --ts sdlcell
cd sdlcell
yarn rw g sdl UserExample
yarn rw g cell UserExample
```

When running the cell generator you'll see this printed:
```
yarn run v1.22.17
$ /Users/tobbe/tmp/sdlcell/node_modules/.bin/rw g cell UserExample
  ✔ Generating cell files...
    ✔ Successfully wrote file `./web/src/components/UserExampleCell/UserExampleCell.mock.ts`
    ✔ Successfully wrote file `./web/src/components/UserExampleCell/UserExampleCell.test.tsx`
    ✔ Successfully wrote file `./web/src/components/UserExampleCell/UserExampleCell.stories.tsx`
    ✔ Successfully wrote file `./web/src/components/UserExampleCell/UserExampleCell.tsx`
  ⠹ Generating types ...
GraphQL Document Validation failed with 1 errors;
  Error 0: GraphQLDocumentError: Cannot query field "userExample" on type "Query". Did you mean "userExamples"?
    at /Users/tobbe/tmp/sdlcell/web/src/components/UserExampleCell/UserExampleCell.tsx:3:5
  ✔ Generating cell files...
    ✔ Successfully wrote file `./web/src/components/UserExampleCell/UserExampleCell.mock.ts`
    ✔ Successfully wrote file `./web/src/components/UserExampleCell/UserExampleCell.test.tsx`
    ✔ Successfully wrote file `./web/src/components/UserExampleCell/UserExampleCell.stories.tsx`
    ✔ Successfully wrote file `./web/src/components/UserExampleCell/UserExampleCell.tsx`
  ✔ Generating types ...
✨  Done in 2.25s.
```

And if you open up the cell in VS Code you'll see errors

![image](https://user-images.githubusercontent.com/30793/158462023-55e45fd4-df9a-4fd7-90c4-64fdd513defc.png)

# Further Details

The issue is that the SDL generator doesn't generate the `userExample` query that the Cell needs. You can "fix"/work around that by running the SDL generator with the `--crud` flag. The SDL generator by default only generates the "find many" query. I think it should generate the "find unique" query too. The "write prisma model" -> "generate sdl" -> "generate cell" workflow is something we should support by default, without any extra flags.

# Fix

I'm more than happy to fix this, as long as we agree to also always generate the "find unique" query.

**Comment by dthyresson** [comment]
@thedavidprice and @Tobbe is the resolution to

1. always crud SDL generation by default?
2. always add the "find" one aka findUnique to the general template

I am ok with 1 to be honest and let people du --crud=false (?) if that is an option to do that.

If so, cli docs here need an update.

**Comment by Tobbe** [comment]
Closing as this is solved by #4785
